frontend/src/features/splReports/columns.tsx

```diff
@@ -0,0 +1,152 @@
+import { ISnapshot } from './interfaces';
```

Comment:
Don't know if this is the correct set of columns to display, but I decided to focus on columns that exist within the snapshot data model as anything from the project will just be based on current project data not snapshot data.

Reply:
Good place to start